morphik.toml

```diff
@@ -118,7 +118,7 @@ model = "ollama_llama"
 enable_entity_resolution = true
 
 [telemetry]
-enabled = true
+enabled = false
```

Comment:
[IMPORTANT] we have telemetry as an opt-out service. that means default should be enabled. To keep the config consistent, please revert this to true when PRing into main.